webui/src/pages/JobsDashboard/JobsTable/JobsTable.tsx

```diff
@@ -40,13 +40,25 @@ function parseData(jobs: Job[]): ParsedJobData[] {
       createdAt: fromTimestamp(job.CreateTime),
       tasks: firstTask,
       jobType: capitalizeFirstLetter(jobType),
-      label: "",
+      label: createLabelArray(job.Labels),
       status: job.State.StateType,
       // action: "Action",
     };
   });
 }
 
+function createLabelArray(label: {[key: string]: string}): string[] {
+  const labelArray: string[] = []
+  for (const [key, value] of Object.entries(label)) {
+    if (value === "") {
+      labelArray.push(key)
+    } else {
+      labelArray.push(value)
+    }
```

Comment:
If the label is `{key: value}` then the key is significant. E.g. if the label is `{least_favourite_food: banana}` it would be bad to just print `banana`. So come up with some way to include both the key and the value when the key is not blank.

Reply:
sure thing!